### DISCUSSION TITLE
extend cpp-libp2p module for micropython

### DISCUSSION THREAD

**Discussion by abuvanth** [micropython/micropython#15712] [Libraries & Drivers]
[libp2p is ](https://docs.libp2p.io/concepts/introduction/overview/)

Please support this module for micropython - https://github.com/libp2p/cpp-libp2p

**Comment by jonnor** [comment]
Hi and thank you for this suggestion. MicroPython modules can be developed by anyone in the community, either as plain Python code, or user C modules (see https://docs.micropython.org/en/latest/develop/cmodules.html).

Those that are interested in libp2p should work together to build such a module. The core MicroPython team cannot develop every single functionality.
